src/Java.md

```diff
@@ -220,4 +220,5 @@ It would be appreciated if there are any Java VTK experts who could convert any
 [AngleWidget2D](/Java/Widgets/AngleWidget2D) | vtkAngleWidget2D | AngleWidget2D Demonstration
 [BalloonWidget](/Java/Widgets/BalloonWidget) |  vtkBalloonWidget | The balloon text describes each object when you hover it.
 [PlaneWidget](/Java/Widgets/PlaneWidget) | vtkPlaneWidget | A finite (bounded) plane that can be interactively placed in a scene.
+[SeedWidget](/Java/Widgets/SeedWidget) | vtkSeedWidget | Places multiple seed points. 
```

Comment:
Change "Places multiple seed points. " to "Places multiple seed points, where a user moves the cursor to and clicks it. "

Reply:
I have updated.